### PR TITLE
Added project filter

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: src/FilterService.Broker/Requests/Interfaces/IRoleService.cs [hunk 1]
```diff
@@ -9,6 +9,6 @@ namespace LT.DigitalOffice.FilterService.Broker.Requests.Interfaces
   [AutoInject]
   public interface IRoleService
   {
-    Task<List<RoleFilteredData>> GetRolesFilterDataAsync(List<Guid> roleIds, List<string> errors);
+    Task<List<RoleFilteredData>> GetRolesFilterDataAsync(List<Guid> rolesIds, List<string> errors);
```
Comment (ndoriomedov) [inline src/FilterService.Broker/Requests/Interfaces/IRoleService.cs]: maybe rename to GetRolesFilteredDataAsync?

File: src/FilterService.Broker/Requests/RoleService.cs [hunk 2]
```diff
@@ -31,21 +31,21 @@ public class RoleService : IRoleService
       _rcGetRoles = rcGetRoles;
     }
 
-    public async Task<List<RoleFilteredData>> GetRolesFilterDataAsync(List<Guid> roleIds, List<string> errors)
+    public async Task<List<RoleFilteredData>> GetRolesFilterDataAsync(List<Guid> rolesIds, List<string> errors)
```
Comment (ndoriomedov) [inline src/FilterService.Broker/Requests/RoleService.cs]: maybe rename to GetRolesFilteredDataAsync?

**Review by ndoriomedov — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

File: src/FilterService.Broker/Requests/Interfaces/IRoleService.cs (hunk 1 above)
Comment (VLiamin) [reply src/FilterService.Broker/Requests/Interfaces/IRoleService.cs]: done

File: src/FilterService.Broker/Requests/RoleService.cs (hunk 2 above)
Comment (VLiamin) [reply src/FilterService.Broker/Requests/RoleService.cs]: done

**Pull request merged**